Cargo.toml

```diff
@@ -164,8 +164,21 @@ escape-html = []
 ## [`Deserializer`]: crate::de::Deserializer
 overlapped-lists = []
 
-## Enables support for [`serde`] serialization and deserialization
-serialize = ["serde"]
+## Enables serialization of some types using [`serde`]. Probably your rarely will
+## need this feature enabled.
+##
+## This feature does NOT provide XML serializer or deserializer. You should use
+## the `serialize` feature for that instead.
+# Cannot name "serde" to avoid clash with dependency.
+# "dep:" prefix only avalible from Rust 1.60
```

Comment:
Does this mean that with edition 2021 / MSRV 1.60, it is possible to simplify all of this?

Reply:
Yes, in Rust 1.60 it is possible to write
```toml
[dependencies]
serde = { version = "1.0", optional = true }

[features]
# depend on the dependency "serde" instead of feature "serde"
# (which will enable "derive" feature of "serde" which is not required
# for the "serialize" feature)
serialize = ["dep:serde"] 

# depend on "serde" dependency with enabled feature "derive"
serde = ["serde/derive"]
```

Reply:
If we don't want to bump the MSRV right now then we should file an issue to simplify this in the future.  I wouldn't be against bumping MSRV though.

Reply:
@Mingun ^

Reply:
You can create an issue if you wish. Anyway, I marked this place with Rust version that is required for more modern solution, so it can be grepped.

I think, this that having more modern solution is not critical, so it is better to avoid bumping MSRV just for it. Otherwise, probably some of our dependents will not happy with that.